src/Fieldtypes/Code.php

```diff
@@ -57,7 +57,6 @@ protected function configFieldItems(): array
                     'xml' => 'XML',
                     'yaml-frontmatter' => 'YAML',
                 ],
-                'width' => 50,
```

Comment:
Why is this line removed?

Reply:
Because the key is defined already in line 32
![2022-02-07_20-47-33](https://user-images.githubusercontent.com/363363/152860964-bbdaa7c0-ff66-4977-a7de-a83b88eec7a2.jpg)